source/manual/alerts/smokey-json-older-than-30m.html.md

```diff
@@ -0,0 +1,38 @@
+---
+owner_slack: "#govuk-2ndline"
+title: smokey.json older than 30m
+section: Development VM
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2019-09-12
+review_in: 6 months
+---
+
+There can be the rare case wherein Smokey has failed to correctly run. While the
+causes can be various and ultimately unknown, the symptoms include a large number
+of Unknown errors in Icinga on the `monitoring` box.
+
+The easiest way to clear these issues is ensure all relevant Smokey tasks are
+killed:
+
+```shell
+#if killall doesn't work kill -15 (or -9) the processes directly
```

Comment:
We might need an example on how to do this / what the command would look like.

Reply:
This should clear it up.